src/Lean/Environment.lean

```diff
@@ -442,13 +488,25 @@ structure Environment where
   private asyncConsts : AsyncConsts := {}
   /-- Information about this asynchronous branch of the environment, if any. -/
   private asyncCtx?   : Option AsyncContext := none
+  /--
+  Realized constants belonging to imported declarations. `none` only from `Environment.ofKernelEnv`,
+  which should never leak into general elaboration.
+  -/
+  private realizedExternConsts? : Option RealizationContext
```

Comment:
Is “extern” the usual term we use here, not “imported”? (Extern may sound like FFI, and at least near env extensions we tend to talk about imported, don't we?)